packages/claudius/claudius.1.2.1/opam

```diff
@@ -0,0 +1,59 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A retro-style graphics library"
+description:
+  "A functional style retro-graphics library for OCaml for building generative art, demos, and games."
+maintainer: [
+  "Michael Dales <michael@digitalflapjack.com>"
+  "Shreya Pawaskar <shreya.pawaskar.main@gmail.com>"
+]
+authors: [
+  "3pleX-dev <afuadajoe@gmail.com>"
+  "Adenuga Israel Abimbola <israelabimmbolaa@gmail.com>"
+  "Michael Dales <michael@digitalflapjack.com>"
+  "Seed Lu <dr.seed.lu@gmail.com>"
+  "Shreya Pawaskar <shreya.pawaskar.main@gmail.com>"
+  "Vanisha1606 <vanishag1606@gmail.com>"
+  "Wah Vanessa <wahvanessa22@gmail.com>"
+]
+license: "ISC"
+tags: ["graphics" "rendering" "paletted"]
+homepage: "https://github.com/claudiusFX/claudius"
+doc: "https://github.com/claudiusFX/claudius"
+bug-reports: "https://github.com/claudiusFX/claudius/issues"
+depends: [
+  "ocaml" {>= "5.1"}
+  "dune" {>= "3.18"}
+  "tsdl" {>= "1.1.0"}
+  "ounit2" {with-test}
+  "odoc" {with-doc}
+  "giflib" {>= "1.1.0"}
+  "imagelib" {>= "20221222"}
+  "crunch" {>= "4.0.0"}
+  "hsluv" {>= "0.1.0"}
+  "ocamlformat" {>= "0.27.0" & with-dev-setup}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/claudiusFX/claudius.git"
+x-maintenance-intent: ["(latest)"]
```

Comment:
```suggestion
x-maintenance-intent: ["(latest)"]
x-ci-accept-failures: ["debian-11"]  # Debian-11 includes ibsdl2-dev.2.0.14 but 2.0.18 is needed to avoid SDL_RenderGeometry error
```
Since we've established the reason for the Debian 11 failure https://github.com/ocaml/opam-repository/pull/28359#issuecomment-3207017405 it could make sense to silence the repeated red flag that it raises :shrug: